appwrite/templates
# Feat: Update starters

## What does this PR do?

- Better copy in starters
- Use dynamic keys in starter

Runtimes to implement:
- [x] Bun
- [x] Dart
- [x] Deno
- [x] Dotnet
- [x] Go
- [x] Kotlin
- [x] Node
- [x] Node-typescript
- [x] PHP
- [x] Python-ML
- [x] Ruby

## Test Plan

Manual QA flow:
- Deploy without scopes
- Try / (200)
- Try /ping (200)
- Check logs (error)
- Deploy with scopes
- Try / (200)
- Check logs (total users)

Runtimes to QA:

Each runtime tested with, and without scopes, and ping endpoint:

![CleanShot 2024-08-05 at 13 14 28@2x](https://github.com/user-attachments/assets/0096a8fc-c7a6-49c3-ad77-5072a62d92b9)

![CleanShot 2024-08-05 at 13 14 46@2x](https://github.com/user-attachments/assets/da8503dd-42bd-4ab3-8f48-fe10ceb7334b)

![CleanShot 2024-08-05 at 17 32 16@2x](https://github.com/user-attachments/assets/ffda2717-46b5-4607-9b6d-e845cbdf1d4d)

- [x] Bun
- [x] Dart
- [x] Deno
- [x] Dotnet
- [x] Go
- [x] Kotlin
- [x] Node
- [x] Node-typescript
- [x] PHP
- [x] Python-ML
- [x] Ruby

## Related PRs and Issues

x

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes
